src/PostVideo/PostVideo.css

```diff
@@ -0,0 +1,16 @@
+.story-video {
+  width: 100%;
+  height: 800px;
+  min-width: 449px;
+}
+.play-icon-container {
+  height: 0;
+}
+.play-icon {
+  position: relative;
+  margin-bottom: -50%;
+  left: 50%;
+  color: #fff;
+  transform: scale(3.5);
+  z-index: 3;
+}
```

Comment:
Why not a CSS module? If you want to keep it like this, you'll have issues since `play-icon` is a common name, and you'll have collisions. With CSS modules, you can use simple names because you're guaranteed to not have collisions.

Reply:
Done